.pre-commit-config.yaml

```diff
@@ -1,6 +1,6 @@
 repos:
-- repo: https://github.com/astral-sh/ruff-pre-commit
-  rev: v0.4.9
-  hooks:
-  - id: ruff
-  - id: ruff-format
+  - repo: https://github.com/astral-sh/ruff-pre-commit
+    rev: v0.4.9
+    hooks:
+      - id: ruff
```

Comment:
Does it make sense to add `args: [--fix]` here?

Reply:
Absolutely, that's kinda the point of using pre-commit isn't it? Although yml formatting here is accidental and doesn't really belong in this PR. I'll extract that and add `--fix`

Reply:
https://github.com/pypa/setuptools/pull/4552